helm/robusta/values.yaml

```diff
@@ -176,6 +176,15 @@ builtinPlaybooks:
   - image_pull_backoff_reporter: {}
 
 # playbooks for non-prometheus based monitoring that use prometheus for enrichment
+- name: "PodEvicted"
+  triggers:
+  - on_pod_evicted: {}
+  actions:
+  - on_pod_evicted_enricher: {}
+  - pod_events_enricher: {}
+  - enrich_pod_with_node_events: {}
+  - logs_enricher: {}
```

Comment:
Regarding the logs, we should test -p mode as we discussed.

Reply:
unfortunately, it didn't work out 
```
unable to retrieve container logs for containerd://0050a65ea07fea153d1e0a6a825d121bbb88680f4a89ed0ab0b9c843f6d6f015
```